backup/backup.go

```diff
@@ -418,7 +418,7 @@ func (bm *BackupManager) backupBinlog(ctx context.Context, op bkop.Operator) err
 	pw2 = nil
 
 	bw := &ByteCountWriter{}
-	key := calcKey(bm.cluster.Namespace, bm.cluster.Name, constants.BinlogFilename, lastBackup.Time.Time)
+	key := calcKey(bm.cluster.Namespace, bm.cluster.Name, constants.BinlogFilename, lastBackup.Time.Time.UTC())
```

Comment:
This is a standalone bug that is worth another issue or PR.
Also, I think the conversion to UTC should happen in `calcKey` because by doing so any use of `calcKey` is protected.

In fact, there is another use of `calcKey` in this same file.

Reply:
Alright. Will create another PR for it, and convert time zone inside `calcKey`.